homeassistant/components/device_tracker/mikrotik.py

```diff
@@ -67,14 +69,29 @@ def __init__(self, config):
     def connect_to_device(self):
         """Connect to Mikrotik method."""
         import librouteros
+        if self.ssl:
+            import ssl
```

Comment:
put this import at top of the file as it's a built-in module.

Reply:
done